docs/dev/adminAPI.md

```diff
@@ -0,0 +1,56 @@
+# Implement your own admin API
+## Why
+The major interest in developing your own admin API is first of all to integrate Work Adventure into an existing computer system.
+To be able to manage the accesses and the rooms be even according to what already exists.
+To do this, you must link the existing user database with Work Adventure.
+In addition, the advantage of implementing your own admin API is to be able to create rooms or access dynamically to Work Adventure.
+
+## Architecture
+[![](https://mermaid.ink/img/pako:eNpdkEFrwzAMhf-K0S4JpJexMDBjkNIVBjuE7VDKvIMbK4tJYgdHXldK__sUklKYTk9PTx9CZ6i8QZDwHfTQiLd35QTXSKcORVG-sgq-RXlX1w-Puc7mdnW0hhp5P_wuhhS5yJfVeJhZOx_awvygoxhwnk21TbbBO0rF02r1LMpEQRnHBoOCVEzOOlnrqk3nBXTmH3W_-1Sw9zGIIx5GS6jg6wbnixlYmN66STPzNisZz_zJvrIhgx5Dr63hD5wnWwE12DNUsnQYKehOgXIXjsbBaMIXY8kHkLXuRsxAR_IfJ1eBpBDxGtpYzef2S-ryBwzWbw8)](https://mermaid-js.github.io/mermaid-live-editor/edit#pako:eNpdkEFrwzAMhf-K0S4JpJexMDBjkNIVBjuE7VDKvIMbK4tJYgdHXldK__sUklKYTk9PTx9CZ6i8QZDwHfTQiLd35QTXSKcORVG-sgq-RXlX1w-Puc7mdnW0hhp5P_wuhhS5yJfVeJhZOx_awvygoxhwnk21TbbBO0rF02r1LMpEQRnHBoOCVEzOOlnrqk3nBXTmH3W_-1Sw9zGIIx5GS6jg6wbnixlYmN66STPzNisZz_zJvrIhgx5Dr63hD5wnWwE12DNUsnQYKehOgXIXjsbBaMIXY8kHkLXuRsxAR_IfJ1eBpBDxGtpYzef2S-ryBwzWbw8)
+<figcaption class="figure-caption text-center">Diagram of the architecture of Work Adventure</figcaption>
+<br>
```

Comment:
In GitHub, you can directly put the Mermaid markdown now :smile: 

````suggestion
```mermaid
graph LR
    style API stroke:#ff475a,stroke-width:2px,stroke: 5 5
    subgraph WorkAdventure
        F(Front) <--> P("Pusher") --> B(Back)
    end
    subgraph YW["Your website"]
        API("Admin API")
        P ---> API
    end
```
````